pkg/markdown/markdown.go

```
@@ -17,69 +16,61 @@ import (
+	),
+	goldmark.WithRendererOptions(
+		html.WithHardWraps(),
+	),
+)
+
+type astTransformer struct{}
 
-func (t *astTransformer) Transform(node *ast.Document, _ text.Reader, _ parser.Context) {
-	t.err = ast.Walk(node, func(n ast.Node, entering bool) (ast.WalkStatus, error) {
+func (astTransformer) Transform(node *ast.Document, _ text.Reader, _ parser.Context) {
+	ast.Walk(node, func(n ast.Node, entering bool) (ast.WalkStatus, error) {
```

golangci-lint (1.62.2)

Error return value of `ast.Walk` is not checked

(errcheck)